econml/ortho_forest.py

```diff
@@ -509,7 +568,8 @@ def second_stage_parameter_estimator_gen(lambda_reg):
         """
         def parameter_estimator_func(Y, T, X,
                                      nuisance_estimates,
-                                     sample_weight=None):
+                                     sample_weight=None,
+                                     X_single=None):
             """Calculate the parameter of interest for points given by (Y, T) and corresponding nuisance estimates."""
```

Comment:
Consider adding a description of X_single to the docstring

Reply:
Added! Thanks!